sdk/appconfiguration/app-configuration/src/internal/tracingHelpers.ts

```diff
@@ -20,6 +20,7 @@ export interface Spannable {
  * @ignore
  */
 export class Spanner<TClient> {
+  // @ts-ignore: Property 'componentName' is declared but its value is never read.
   constructor(private baseOperationName: string, private componentName: string) {}
```

Comment:
if componentName is never read, why do we have it on the class?

Reply:
I don't think we do! Though should I remove it as part of this PR? I tried to minimize the changes, but I could go and do some of these refactors too.

Reply:
I will remove it!